Runtimes/Core/cmake/modules/DefaultSettings.cmake

```
@@ -41,6 +41,7 @@ if(APPLE)
   set(SwiftCore_ENABLE_RUNTIME_OS_VERSIONING ON)
   set(SwiftCore_ENABLE_OVERRIDABLE_RETAIN_RELEASE ON)
   set(SwiftCore_THREADING_PACKAGE "DARWIN")
+  set(SwiftCore_ENABLE_PRESPECIALIZATION ON)
```

This will prevent the user to change the default in the cache
```suggestion
  set(SwiftCore_ENABLE_PRESPECIALIZATION_default ON)
```